app/helpers/image_service_helper.rb

```diff
@@ -82,7 +104,7 @@ def member_image_tag(parent_id:, member:, size_key: nil, lazy: false, use_image_
 
   private
 
-  def create_riiif_url(path)
+  def create_iiif_url(path)
     if CHF::Env.lookup(:public_riiif_url)
       url = Addressable::URI.parse(CHF::Env.lookup(:public_riiif_url))
```

Comment:
I stopped short of this so we weren't spending too much time changing our config requirements in local_env.